learning/katas/go/Core Transforms/Map/ParDo OneToMany/task.md

```diff
@@ -0,0 +1,32 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# ParDo - One to Many
+
+In the previous kata we learned that ParDo maps a single element into another element.
+In this kata we will map a single element into many by splitting a sentence into words.
+
+**Kata:** Please write a ParDo that maps each input sentence into words tokenized by whitespace (" ").
+
+<div class="hint">
+  Use <a href="https://godoc.org/github.com/apache/beam/sdks/go/pkg/beam#ParDo">
+  ParDo</a>
```

Comment:
Consider using the qualifed import name, like they'd see in their Go code (eg. beam.ParDo) , rather than just the single method. It would look odd to other languages, but the generally explicit package/provenance of identifiers is a hallmark of Go.

Reply:
@lostluck Do you mean this?  If so, I could check across the various tasks of the existing katas to make sure its consistent.
```
<div class="hint">
  Use <a href="https://godoc.org/github.com/apache/beam/sdks/go/pkg/beam#ParDo">
  beam.ParDo</a>
```

Reply:
I do!

Reply:
Rather, I do. But I can merge this in as is and the change can happen in another PR or it can happen this PR. Whichever you prefer. 

Sorry for the delay in getting back to this.

Reply:
I created [BEAM-9928](https://issues.apache.org/jira/browse/BEAM-9928) so this is not forgotten.  My vote is to merge the request.